[MSCRIPTING-9] - basic java scripting support (#5)
apache/maven-scripting-plugin: README.md

 Optional supplemental description.
```
+ Make sure you have added the necessary tests (JUnit/IT) for your changes.

> + Run all the tests with `mvn -Prun-its verify` to assure nothing else was accidentally broken (you can need to `mvn install` before).

Reviewer: ```suggestion
+ Run all the tests with `mvn -Prun-its verify` to assure nothing else was accidentally broken (you may need to run `mvn install` beforehand).
```

> + Run all the tests with `mvn -Prun-its verify` to assure nothing else was accidentally broken (you can need to `mvn install` before).

Reviewer: Hint: It should *never* need an `install`!

Author: @bmarwell well I will not enter into the debate but this is not my change but the way the project was setup, invoker+mrm config does not seem to work reliably without an install so added the comment.

> + Run all the tests with `mvn -Prun-its verify` to assure nothing else was accidentally broken (you may need to run `mvn install` beforehand).

Reviewer: Review hint: This was not introduced by @rmannibucau’s change, mrm+invoker was already broken before.
